### ISSUE TITLE
Add Plugin... > Rescan doesn't detect newly installed LV2 plugins

### ISSUE THREAD

**Issue by maxigaz** [rncbc/qtractor#344]
If I have Qtractor 0.9.26.34git.676c3f running, install an LV2 plugin through Arch Linux's package manager, and choose Add Plugin... > Rescan, the plugin doesn't show up in the list. I need to restart Qtractor so that the plugin appears in the list.

In the options, everything on the Plugins tab has the default settings applied.

**Comment by rncbc** [comment]
yes, it's true, on the very special case of LV2 plugins, the "Rescan" doen't work as such: you'll need to close and restart the session anyway, for new LV2 plugins to get picked up.

**Comment by danzankul** [comment]
[No instruments on Qtractor]

I'd like to add instruments sets on Qtractor (linux mint 22.0) but I don't know how to, since import instruments windows leads me to an empty directory asking for ins *.lf2-3 midnam files.
